src/openshift/project.ts

```diff
@@ -22,13 +22,22 @@ export class Project extends OpenShiftItem {
     }
 
     static async del(context: OpenShiftObject): Promise<string> {
```

Comment:
This command is perfectly fine, it is always called from context menu in OpenShift App Explorer and never from Commend Palette, so there is no need to change it. What you want to do is to create new command 
``` javascript
static async delProject(): Promise<string> {
// collect all the required info here
// get instance of OpenShiftObject being deleted
// call static del function with this instance
}
```